plasticos_web_lead_ai_triage/models/web_lead_triage.py

```
@@ -0,0 +1,689 @@
+                    Attachment.create({
+                        "name": fname,
+                        "type": "binary",
+                        "datas": base64.b64encode(content).decode("ascii"),
+                        "res_model": "plasticos.intake",
+                        "res_id": self.intake_id.id,
+                        "mimetype": content_type,
+                    })
+
+                _logger.info("Attached image %s to web lead %s.", fname, self.lead_id)
+
+            except Exception as exc:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)